NetCoreRestApi/WebAPI/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using BusinessLayer.Manager;
+using Common.Converter;
+using DataLayer.Models;
+using Microsoft.AspNetCore.Mvc;
+using ServiceLayer.DataTransferObjects;
+
+namespace WebAPI.Controllers
+{  
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoriesController : ControllerBase
+    {
+        private IConverter<CategoryDTO, CategoryModel> _converter;
+        private ICategoryManager _categoryManager;
+
+        public CategoriesController(IConverter<CategoryDTO, CategoryModel> converter, ICategoryManager categoryManager)
```

Comment:
first should be manager interfaces then converters

Reply:
done